packages/synapse-react-client/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,87 @@
+import { Box, Stack, Typography } from '@mui/material'
+
+export type HeaderProps = {
+  backgroundImage: string
```

Comment:
Since this is just passed along as a CSS prop, the name can be more generic
```suggestion
  backgroundCss: string
```